libraries/AP_GPS/AP_GPS.h

```diff
@@ -457,6 +473,28 @@ class AP_GPS
 
     // inject data into all backends
     void inject_data_all(const uint8_t *data, uint16_t len);
+
+    // GPS blending and switching
+    Vector2f _NE_pos_offset_m[GPS_MAX_RECEIVERS]; // Filtered North,East position offset from GPS instance to blended solution in _output_state.location (m)
+    float _hgt_offset_cm[GPS_MAX_RECEIVERS]; // Filtered height offset from GPS instance relative to blended solution in _output_state.location (cm)
+    Vector3f _blended_antenna_offset; // blended antenna offset
+    float _blend_weights[GPS_MAX_RECEIVERS]; // blend weight for each GPS. The blend weights must sum to 1.0 across all instances.
+    Location _output_location_corrected; // output location after application of the  filtered offset corrections _NE_pos_offset_m and _hgt_offset_cm
```

Comment:
_hgt_offset_cm is used internally to calculate corrected_location[i].alt
_blended_antenna_offset is published by get_antenna_offset()
_blend_weight is used internally in many places

_output_location_corrected is not being used and will be removed.